pkgs/by-name/mu/muse-sounds-manager/package.nix

```diff
@@ -0,0 +1,76 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  autoPatchelfHook,
+  dpkg,
+  fontconfig,
+  zlib,
+  icu,
+  libX11,
+  libXext,
+  libXi,
+  libXrandr,
+  libICE,
+  libSM,
+  openssl,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "muse-sounds-manager";
+  version = "1.1.0.587";
+
+  # Use web.archive.org since the upstream does not provide a stable
+  # (versioned) URL.  To see if there are new versions, visit
+  # http://web.archive.org/cdx/search/cdx?url=https://muse-cdn.com/Muse_Sounds_Manager_Beta.deb
+  # then replace the date in the URL below with date when the SHA1
+  # changes (currently A3NX3WHFZWXCHZVME2ABUL2VRENTWOD5) and replace
+  # the version above with the version in the .deb metadata (or in the
+  # settings of muse-sounds-manager).
+  src = fetchurl {
+    url = "https://web.archive.org/web/20240826143936/https://muse-cdn.com/Muse_Sounds_Manager_Beta.deb";
+    hash = "sha256-wzZAIjme1cv8+jMLiKT7kUQvCb+UhsvOnLDV4hCL3hw=";
+  };
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    dpkg
+  ];
+
+  buildInputs = [
+    fontconfig
+    stdenv.cc.cc
+    zlib
+  ] ++ runtimeDependencies;
+
+  runtimeDependencies = map lib.getLib [
+    icu
+    libX11
+    libXext
+    libXi
+    libXrandr
+    libICE
+    libSM
+    openssl
+  ];
+
+  unpackPhase = "dpkg -x $src .";
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out
+    mv usr/* opt $out/
+    substituteInPlace $out/bin/muse-sounds-manager --replace-fail /opt/ $out/opt/
+
+    runHook postInstall
+  '';
+
+  meta = {
+    description = "Manage MuseScore Libraries";
```

Comment:
a small thing but i think it is worth a change.
maybe the description could be a bit clearer as:
"Manage Muse Sounds libraries for MuseScore" as "MuseScore Libraries" can mean a library of music scores and not instument sounds.

Reply:
Thanks! I've copied that description from .deb, but it is really not clear. Done!

Reply:
Sorry, I just realized that adding the word Hub to the desccription would be a good idea as well.
As some might search for Muse Hub because they don't know the name for the linux version.
So maybe 

"Manage Muse Hub Sounds libraries for MuseScore"

Thanks for listening. :heart:

Reply:
It seems that MuseScore authors use "Muse Sounds" as the name of the library: https://musescore.org/en/handbook/4/installing-muse-sounds . "Muse Hub Sounds" would be a non-existent name. I could change the description to "Manage Muse Sounds (Muse Hub) libraries for MuseScore"

Reply:
Done